src/candela/VisComponent/README.md

```diff
@@ -73,17 +73,31 @@ the constructor for specific subclasses of VisComponent.
   This includes any data, visual matchings, or other settings pertinent to the
   visualization. Options are specified in the form `{name: value}`.
 
+**Note**: The constructor for the abstract superclass is empty. You should use
+the constructor for specific subclasses of `VisComponent`.
+
+## component.render()
+
+Renders the component to it's container using the current set of options.
+
+**Note**: The `VisComponent` `render()` method simply throws
+an exception; if you truly want your component to do nothing when it renders,
+simply redefine the method to be a no-op.
+
 ## component.serializationFormats
 
 The `serializationFormats` field is a list of strings of supported formats.
 Formats include:
 
-* `'png'`: A base64-encoded string for a PNG image. This string may be placed in the
+* `'png'`: A base64-encoded string for a PNG image. The string may be placed in the
 `src` attribute of an `<img>` element to show the image.
 
-* `'svg'`: A base64-encoded string for an SVG scene. This string may be placed in the
+* `'svg'`: A base64-encoded string for an SVG scene. The string may be placed in the
 `src` attribute of an `<img>` element to show the image.
 
 ## component.serialize(format)
 
 Serializes the component into the specified **format**.
+
+**Note**: [`render()`](#componentrender) must be called at least once
+before serialize() is called.
```

Comment:
Is this true in general? I know it's true for vcharts components, but it seems like that requirement is something that should be mentioned in the specific docs for those components.